### PR TITLE
add proton-ge derivation

### PR DESCRIPTION
**Pull request by NotAShelf**
Simply adds the proton-ge derivation to nix-gaming flake's outputs, I was getting tired of maintaining in my own flake.

[This issue](https://github.com/NixOS/nixpkgs/issues/73323#issuecomment-1079939987) provides instructions on how the user may use the proton-ge derivation on primarily NixOS systems. I assume it would be similar on extending PATH on non-nixos, or linking imperatively.

### CODE REVIEW

**Comment by NotAShelf** [comment]
Note: I kept the passthru script for update, but could or probably should be removed.

**Comment by fufexan** [comment]
Alright, could you add the package to CI as well? And revert the formatting done on `npins/default.nix`.

**Comment by NotAShelf** [comment]
should be good now

**Comment by NotAShelf** [comment]
P.S. I'm aware that proton-ge has received an update while this PR was still ongoing. I'll bump it through a separate PR possibly on the weekend.

File: pkgs/proton-ge/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+stdenv.mkDerivation (finalAttrs: {
+  name = "proton-ge-custom";
+  version = "GE-Proton8-2";
+
+  src = fetchurl {
+    url = "https://github.com/GloriousEggroll/proton-ge-custom/releases/download/${finalAttrs.version}/${finalAttrs.version}.tar.gz";
+    sha256 = "sha256-gof4yL5sHPKXDC4mDfPyBIvPtWxxxVy6gHx58yoTEbQ=";
+  };
+
+  passthru.updateScript = writeScript "update-pytr" ''
```
Comment (Shawn8901) [inline pkgs/proton-ge/default.nix:16]: Likely you don't want to name the update script like that, but something like `update-proton-ge`.

I don't know if you got that error from an older version of my flake package (but as you Name pytr, which is a python module i also have added as a package)

**Comment by NotAShelf** [comment]
> > P.S. I'm aware that proton-ge has received an update while this PR was still ongoing. I'll bump it through a separate PR possibly on the weekend.
> 
> Why not update this PR directly?

I'm doing this on github web, lmao.

File: pkgs/proton-ge/default.nix (hunk 1 above)
Comment (NotAShelf) [reply pkgs/proton-ge/default.nix:16]: I think I got the package from your dots, it was linked to me a few months ago when I was new to NixOS.

To be honest, I am thinking of dropping the updatescript as a whole. It's not hard to update one derivation manually every few weeks.

**Pull request merged**